package lyd.ai.dataflow.functions;

public class SuccessException extends Exception {
}
